controller/src/main/java/io/pravega/controller/server/rest/RESTServer.java

```diff
@@ -68,12 +74,37 @@ protected void startUp() {
         long traceId = LoggerHelpers.traceEnterWithContext(log, this.objectId, "startUp");
         try {
             log.info("Starting REST server listening on port: {}", this.restServerConfig.getPort());
-            httpServer = GrizzlyHttpServerFactory.createHttpServer(baseUri, resourceConfig, true);
+            if (restServerConfig.isTlsEnabled()) {
+                SSLContextConfigurator sslCon = new SSLContextConfigurator();
```

Comment:
`Con` is supposed to be context? Can we be more descriptive, please?

Reply:
Fixed. Renamed it to be more descriptive.